packages/block-library/src/embed/variations.js

```diff
@@ -216,6 +217,15 @@ const variations = [
 		patterns: [ /^https?:\/\/(www\.)?mixcloud\.com\/.+/i ],
 		attributes: { providerNameSlug: 'mixcloud', responsive: true },
 	},
+	{
+		name: 'pocketcasts',
+		title: 'Pocket Casts',
+		icon: embedPocketCastsIcon,
+		keywords: [ __( 'pocketcasts' ), __( 'podcast' ), __( 'audio' ) ],
```

Comment:
this should be unnecessary if it's in the name

Reply:
Fixed!